nextflow_schema.json

```
@@ -63,6 +63,10 @@
                     "enum": ["higlass", "pretext", "both"],
                     "default": "both"
                 },
+                "btk_location": {
+                    "type": "string",
```

If it's given, it must be a directory.
There's also `"exists": true` which'd be nice to use, but I wonder how it works for optional values.
```suggestion
                    "type": "string",
                    "format": "directory-path",
```

---

I'd expect that `exists` will try and validate the null value given by default. I agree with adding the `format` though.